From 8d3ca4b9e1ae28311254b246b8eb85e79d0e7cca Mon Sep 17 00:00:00 2001
From: C-ty <cty222@hotmail.com>
Date: Wed, 24 Jul 2013 09:43:38 -0700
Subject: [PATCH 3/5] fix create_patch.sh

---
 create_patch.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/create_patch.sh b/create_patch.sh
index 2ba2c8d..c3d5b2e 100755
--- a/create_patch.sh
+++ b/create_patch.sh
@@ -1 +1 @@
-git format-patch 1fbb57374cccbef720200f463ee6ebe84660e1fa -1 -o /usr/local/AChenHow_kernel_patch/ 
+git format-patch b699fe4aa7097fadfa2d96a6677084b140d7a7ad -o /usr/local/AChenHow_kernel_patch/ 
-- 
1.7.9.5

